client/signup/steps/design-type-with-atomic-store/index.jsx

```diff
@@ -154,6 +174,7 @@ class DesignTypeWithAtomicStoreStep extends Component {
 
 		return (
 			<div className="design-type-with-atomic-store__substep-wrapper">
+				{ ! countryCode && <QueryGeo /> }
```

Comment:
Not sure if we want to query the Geo endpoint for all the users coming to this page – might be a ton of traffic. I know we could preload it this way, but what if we waited until user decides to click on the "Start with an online store"? If it's like a 1-2 seconds delay to get the Geo data, I think it would be okay.

/cc @jhnstn

Reply:
Fixed: I now query the Geo endpoint only after "store" is clicked. It leads to a delay (the geo data is not preloaded and is requested only after the click), but saves some API traffic for logged-out users.